app/uk/gov/hmrc/claimvatenrolmentfrontend/featureswitch/frontend/config/FeatureSwitchProviderConfig.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.claimvatenrolmentfrontend.featureswitch.frontend.config
+
+import play.api.Configuration
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
+import uk.gov.hmrc.claimvatenrolmentfrontend.featureswitch.frontend.models.FeatureSwitchProvider
+
+import javax.inject.{Inject, Singleton}
+
+@Singleton
+class FeatureSwitchProviderConfig @Inject()(configuration: Configuration) {
+
+  val servicesConfig = new ServicesConfig(configuration)
+
+  lazy val selfBaseUrl: String = servicesConfig.baseUrl("self")
+
+  lazy val selfFeatureSwitchUrl = s"$selfBaseUrl/claim-vat-enrolment /test-only/api/feature-switches"
+
+  lazy val claimVatEnrolmentFeatureSwitchUrl =
```

Comment:
As CVE doesn't have a backend service we don't need claimVatEnrolmentFeatureSwitchUrl and claimVatEnrolmentFeatureSwitchProvider. So featureSwitchProviders will just include the Seq(selfFeatureSwitchProvider)